custom_components/frigate/sensor.py

```diff
@@ -290,6 +290,9 @@ def state(self) -> float | None:
                 .get("gpu")
             )
             if data is not None:
+                if data == -1:
```

Comment:
How about we check for the type instead (cover more cases that will break), and unindent this (readability), i.e. something like:

```py
if data is None or !isinstance(data, str):
  return None
try
  [...stuff...]
```

Reply:
Yeah makes sense